unlock-app/src/services/walletService.js

```diff
@@ -73,8 +75,10 @@ export default class WalletService extends EventEmitter {
     this.web3 = new Web3(provider)
 
     const networkId = await this.web3.eth.net.getId()
-    if (unlockAddress) {
-      this.unlockContractAddress = Web3Utils.toChecksumAddress(unlockAddress)
+    if (this.unlockAddress) {
```

Comment:
I think it is strange that we have both `this.unlockAddress` and `this.unlockContractAddress`? I think we only use the latter one. Could we just maybe move this part to the constructor and not use `this.unlockAddress` anymore?